taipy/core/scenario/_scenario_manager.py

```diff
@@ -468,3 +468,25 @@ def _get_by_config_id(cls, config_id: str, version_number: Optional[str] = None)
         for fil in filters:
             fil.update({"config_id": config_id})
         return cls._repository._load_all(filters)
+
+    @classmethod
+    def _get_by_creation_time(
+        cls, start_time: datetime, end_time: datetime, version_number: Optional[str] = None
+    ) -> List[Scenario]:
+        """
+        Get all scenarios by a given creation time period.
+        The time period is inclusive.
+
+        Parameters:
+            start_time (datetime): Start time of the period.
+            end_time (datetime): End time of the period.
```

Comment:
I would call them `from` and `to`.

Reply:
`from` is a Python keyword though, so we can't use it

Reply:
Sure. Let's keep your proposal, then.